inst/include/interface/rcpp/rcpp_objects/rcpp_tmb_dnorm_distribution.hpp

```diff
@@ -12,33 +12,6 @@
 #include "../../../distributions/distributions.hpp"
 #include "rcpp_interface_base.hpp"
 
-/**
```

Comment:
Looks like these lines were deleted but not added to a unique file?

Reply:
I decided (for now, not sure what is the best way) to put all of the interfaceBase classes in rcpp_interface_base.hpp so that we don't have a ton of files, so the `DistributionsInterfaceBase` class definition is now copied over to line 116 of the rcpp_interface_base file.